src/Tribe/Settings.php

```diff
@@ -864,9 +879,16 @@ public function add_child_tabs_to_nav( Tribe__Settings_Tab $tab ) {
 	 * @since TBD
 	 */
 	protected function get_modal_controls(): void {
+		$current_tab = $this->get_tab( $this->get_current_tab() );
+		if ( empty( $current_tab ) ) {
+			return;
+		}
```

Comment:
Just a note that this is the second time I've seen this same block. It might not be able to be abstracted, but if it can be that might be a good thing to pull out.

Reply:
I think I removed the other use of it but good to point out!
FYI: My habit is to abstract once I hit 3 uses.